variants/PNUCLEO_WB55RG/otp.h

```diff
@@ -0,0 +1,75 @@
+/**
+  ******************************************************************************
+  * @file    otp.h
+  * @author  MCD Application Team
+  * @brief   OTP manager  interface
+  ******************************************************************************
+   * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2019 STMicroelectronics. 
+  * All rights reserved.</center></h2>
+  *
+  * This software component is licensed by ST under BSD 3-Clause license,
+  * the "License"; You may not use this file except in compliance with the 
+  * License. You may obtain a copy of the License at:
+  *                        opensource.org/licenses/BSD-3-Clause
+  *
+  ******************************************************************************
+ */
+
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __OTP_H
+#define __OTP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+  /* Includes ------------------------------------------------------------------*/
+/* ----------------------------------- *
+ *  Packed usage (compiler dependent)  *
+ * ----------------------------------- */
+#undef PACKED__
+#undef PACKED_STRUCT
+
+#if defined ( __CC_ARM )
+  #if defined (__ARMCC_VERSION) && (__ARMCC_VERSION >= 6010050U)
+    #define PACKED__ __attribute__((packed))
+    #define PACKED_STRUCT struct PACKED__
+  #else
+    #define PACKED__(TYPE) __packed TYPE
+    #define PACKED_STRUCT PACKED__(struct)
+  #endif
+#elif defined   ( __GNUC__ )
+  #define PACKED__ __attribute__((packed))
+  #define PACKED_STRUCT struct PACKED__
+#elif defined (__ICCARM__)
+  #define PACKED_STRUCT __packed struct
+#elif
+  #define PACKED_STRUCT __packed struct
+#endif
+
+  /* Exported types ------------------------------------------------------------*/
+  typedef  PACKED_STRUCT
+  {
```

Comment:
No need to define this as Arduino only manage one toolchain.
Moreover the packed attribute does not need to be redefined. It is already managed here:
https://github.com/stm32duino/Arduino_Core_STM32/blob/bee856121e7406314226506d20617b523a092a70/system/Drivers/STM32WBxx_HAL_Driver/Inc/stm32wbxx_hal_def.h#L115

```suggestion
  typedef struct __packed {
```